Circle.cpp

```diff
@@ -1,14 +1,37 @@
 #ifndef Circle_cpp
 #define Circle_cpp
+
+#define _USE_MATH_DEFINES
+
 #include <iostream>
-#include "Draw.cpp"
+#include <math>
+
+#include "Shape.cpp"
+#include "Area.cpp"
+#include "Input.cpp"
+
 using namespace std;
-class Circle : public Draw
+class Circle : public Shape , public Input
 {
 public:
     void drawShape() override
     {
-        cout << "circle" << endl;
+        cout << "circle" << endl; 
+    }
+    int inputRadius() override 
+    {
+        int radius;
+        cout << "Enter Circle Radius" << endl;
+        cin >> radius;
+        return radius;
+    }
+    int CalculateArea() override
```

Comment:
PI or `M_PI` is a floating point data type. Converting/casting it to an `int` will almost always result in data loss.

Reply:
Yeah we can change its data type as float or double but as i mentioned in the video and voice note that first we have to resolve the error that is occurring with math.h library

Reply:
You are already using `M_PI` from math library. What's there to resolve? Are you saying you submitted a PR which doesn't compile?

Besides it's not a question of first. The two (math library and floating point data type) are completely separate problems. You need to use the correct data type from the start regardless of other integration problems (related or unrelated).